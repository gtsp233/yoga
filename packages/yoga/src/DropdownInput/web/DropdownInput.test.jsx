import React from 'react';
import { render } from '@testing-library/react';

import { FlagsIcons } from '@gympass/yoga-icons';
import { ThemeProvider, DropdownInput } from '../..';

describe('<DropdownInput />', () => {
  it.only('should match snapshot', () => {
    const { container } = render(
      <DropdownInput
        placeholder="Company Name"
        selectedCountry={{
          icon: FlagsIcons.FlagBrazil,
          name: 'Brazil',
          id: 'BR',
        }}
      />,
      { wrapper: ThemeProvider },
    );

    expect(container).toMatchSnapshot();
  });
});
