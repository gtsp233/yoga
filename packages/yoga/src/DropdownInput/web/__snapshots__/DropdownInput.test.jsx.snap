// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownInput /> should match snapshot 1`] = `
.c3 {
  margin-right: 8px;
  margin-left: 16px;
  width: 20px;
  height: 20px;
}

.c4 {
  margin-right: 12px;
  width: 16px;
  height: 16px;
}

.c0 {
  position: relative;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-radius: 8px;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #fff;
  border: 1px solid #ccc;
  box-sizing: border-box;
  background: #fff;
  height: 52px;
  margin: 8px 0 0;
  -webkit-transition: all 0.1s ease-in-out;
  transition: all 0.1s ease-in-out;
  width: 320px;
}

.c7 {
  background: none;
  border: none;
  cursor: pointer;
  display: none;
  outline: none;
  padding: 20px;
}

.c6 {
  background: none;
  border: none;
  color: #231B22;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  font-family: Rubik;
  font-size: 14px;
  font-style: normal;
  font-weight: 400;
  height: 100%;
  -webkit-letter-spacing: 0px;
  -moz-letter-spacing: 0px;
  -ms-letter-spacing: 0px;
  letter-spacing: 0px;
  line-height: 20px;
  outline: none;
  text-align: left;
  padding-left: 16px;
}

.c6::-webkit-input-placeholder {
  color: #6B6B78;
}

.c6::-moz-placeholder {
  color: #6B6B78;
}

.c6:-ms-input-placeholder {
  color: #6B6B78;
}

.c6::placeholder {
  color: #6B6B78;
}

.c5 {
  width: 1px;
  height: 28px;
  background: #D7D7E0;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: auto;
  background: none;
  border: none;
  outline: none;
  cursor: pointer;
  padding: 0;
  height: 100%;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <button
        class="c2"
        id="button-drop-down"
        type="button"
      >
        <div
          class=""
        >
          <svg
            class="c3"
            height="20"
            marginLeft="4"
            marginRight="2"
            width="20"
          />
        </div>
        <svg
          class="c4"
          cursor="pointer"
          height="16"
          mr="xsmall"
          stroke="#6B6B78"
          width="16"
        />
        <div
          class="c5"
        />
      </button>
      <input
        class="c6"
        data-testid="input"
        placeholder="Company Name"
        value=""
      />
      <button
        class="c7"
        type="button"
      >
        <svg
          fill="#d8385e"
          height="15"
          width="15"
        />
      </button>
    </div>
  </div>
</div>
`;
